cmd/producer/main_test.go

```diff
@@ -44,37 +44,21 @@ func TestAsyncRequestHeader(t *testing.T) {
 		returncode       int
 	}{{
 		name:       "async get request",
-		async:      true,
 		method:     "GET",
 		body:       "",
 		returncode: 202,
-	}, {
-		name:       "non async get request",
-		async:      false,
-		method:     "GET",
-		body:       "",
-		returncode: 200,
-	}, {
-		name:       "non async post request",
-		async:      false,
-		method:     "POST",
-		body:       `{"body":"this is a body"}`,
-		returncode: 200,
 	}, {
 		name:       "async post request with too large payload",
-		async:      true,
 		method:     "POST",
 		body:       `{"body":"this is a larger body"}`,
 		returncode: 500,
```

Comment:
s/500/http.StatusInternalServerError/g in the file.

Reply:
thanks!